lib/routers/task/index.js

```diff
@@ -0,0 +1,63 @@
+const { Router } = require('express');
+const router = Router({ mergeParams: true });
+
+router.param('taskId', (req, res, next, taskId) => {
+  return Promise.resolve()
+    .then(() => {
+      req.taskId = taskId;
+      next();
+    });
+});
+
+router.get('/:taskId', (req, res, next) => {
+  const params = { action: 'read' };
+
+  return req.workflow(params, `/${req.taskId}`)
+    .then(response => {
+      res.response = response.json.data;
+      next();
+    })
+    .catch(next);
+});
+
+router.put('/:taskId', (req, res, next) => {
```

Comment:
We might want to make this explicit to the status endpoint, to avoid confusion with actually updating the body of the case.

Reply:
Done.